syncs/syncs.go

```diff
@@ -192,6 +192,22 @@ func (m *Map[K, V]) LoadOrStore(key K, value V) (actual V, loaded bool) {
 	return actual, loaded
 }
 
+// LoadOrInit returns the value for the given key if it exists, otherwise f is
+// called to construct the value to be set.
+func (m *Map[K, V]) LoadOrInit(key K, f func() V) (actual V, initialized bool) {
```

Comment:
The returned `bool` is the opposite of `LoadOrStore` and `LoadAndDelete`. I think this should be "loaded" instead to be consistent with the other methods and also to emphasize the "Load" is the primary verb here.

In fact, L202 seems to match the "loaded" semantic, but L208 matches the "initialized" semantic.